content/en/apps/concepts/interoperability.md

```diff
@@ -3,36 +3,40 @@ title: Interoperability
 linkTitle: "Interoperability"
 weight: 8
 description: >
-  Exchanging information between CHT Core and other systems
+  Exchanging information between the CHT Core and other health systems 
 keywords: interoperability integrations fhir icd openhie openhim
 relatedContent: >
   apps/features/integrations
 ---
 
-# Introduction
+## Introduction 
 
-Interoperability is the ability of health information systems to work together, even if they weren't specifically designed to work together. With interoperability, patient information can be seen, exchanged, and used across different platforms. This is different from _integration_ which requires custom development to connect two specific systems together. 
+Interoperability refers to the ability of different health information systems and applications to communicate with each other and exchange data seamlessly. With interoperability, patient information can be seen, exchanged, and used across different platforms. The information/data exchanged has to be understood across the different software for these systems to become interoperable. This is different from _integration_ which requires custom development to connect two specific systems together.
 
-Interoperability is the best practice for health systems because it allows information from one system to be shared with one or more other systems with no additional development.
+Interoperability is the best practice for health systems because it allows information from one system to be shared with one or more other systems with no additional development. Interoperability allows technical teams to scale in an efficient and repeatable manner due to the already predefined standards. 
 
-# Components
+## CHT Interoperability
 
-Useful components and reference information for interoperability include:
+The native CHT database structure does not map directly to a [Fast Healthcare Interoperability Resources (FHIR)](http://www.hl7.org/fhir/) message format. To be compatible, we use a middleware to convert the CHT data structure into a standardized JSON format so the other systems can read it.
 
-- [OpenHIE](https://ohie.org/) defines the architecture for an interoperability layer.
-- [OpenHIM](http://openhim.org/) is a middleware component designed to ease interoperability between systems.
-- [HL7 FHIR](https://www.hl7.org/fhir/index.html) is a messaging format to allow all systems to understand the format of the message.
-- [ICD-11](https://www.who.int/standards/classifications/classification-of-diseases) and [LOINC](https://loinc.org/) are examples of classification systems.
+{{< figure src="flow.png" link="flow.png" >}}
```

Comment:
Let's use the new mermaid chart Josh added!

I'd make this a suggestion, but GH isn't playing nice with me having a code block inside my suggestion block :shrug: 

Here's the code to uses , but the first line needs to wrapped in open ` ```mermaid` and close  ` ``` ` :

```
graph LR
cht[CHT]
mediator_a([Mediator])
mediator_b([Mediator])
openhim[OpenHIM]
cht -- Outbound push\nfa:fa-arrow-right --- mediator_a
cht -- API request\nfa:fa-arrow-left --- mediator_b
mediator_a -- Request\nfa:fa-arrow-right --- openhim
mediator_b -- Channel\nfa:fa-arrow-left --- openhim
```

Reply:
Thanks, adding the mermaid.